### DISCUSSION TITLE
米家体脂秤s400无实体

### DISCUSSION THREAD

**Discussion by zraxx** [XiaoMi/ha_xiaomi_home#400] [Q&A]
![Screenshot_2024-12-23-23-56-04-051_io.homeassistant.companion.android.png](https://github.com/user-attachments/assets/244e6325-7fc5-423b-86a8-54b3ae858559)

**Comment by SusanPhevos** [comment]
[yunmai.scales.ms103](https://home.miot-spec.com/spec/yunmai.scales.ms103) 的 MIoT-Spec-V2 大部分是非标实例，需要取消勾选 xiaomi_home > 设置 > 隐藏非标准生成实体 才能显示所有实体，就像[这样](https://github.com/XiaoMi/ha_xiaomi_home/issues/362#issuecomment-2717818917)。
